src/kasse.c

```diff
@@ -112,177 +112,75 @@ static void print_screen(void) {
 static void print_log(char *name, int item_price, int einheiten, char *nickname,
                       char *rest) {
   char *time = get_time();
-  char price[10];
-  /* Format:
-     Transaction-ID (Anzahl verkaufter Einträge, inklusive des zu druckenden!)
-     -- 6-stellig
-     Uhrzeit -- 8-stellig
-     Eintragname (= Getränk) -- 9-stellig
-     Preis (in Cents) -- 9-stellig
-     Anzahl -- 2-stellig
-     Nickname (falls es vom Guthaben abgezogen wird) -- 10-stellig
-     restguthaben (9-stellig)
-
-     + 7 leerzeichen
-     --> 48 zeichen
-     */
-  if (format_euro(price, 10, item_price) == NULL) {
+  /* TODO program a custom character for the eurosign.
+   * Until then, we just overwrite it with "EUR" */
```

Comment:
I don’t quite follow. Is EURSYM (defined as \xA2) good to use or not? If no, don’t we need to update EUR_FORMAT and its corresponding size constant as well?

Reply:
The printer doesn't know what to do with this byte.
Since we will throw that code away after the binlog patch, I didn't bother much with it.

Reply:
I suggest nothing that in the comment, e.g. “TODO: teach the euro sign to the printer. Until then, …”